<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartScreen</class>
 <widget class="QDialog" name="StartScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="LoginForm" name="pageLogin"/>
   <widget class="RegistrationForm" name="pageRegistration"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginForm</class>
   <extends>QWidget</extends>
   <header>loginform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RegistrationForm</class>
   <extends>QWidget</extends>
   <header>registrationform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
